src/Polly/RateLimit/LockFreeTokenBucketRateLimiter.cs

```diff
@@ -15,7 +15,7 @@ internal sealed class LockFreeTokenBucketRateLimiter : IRateLimiter
     private long addNextTokenAtTicks;
 
 #if !NETSTANDARD2_0
-    private SpinWait spinner = default;
+    private static SpinWait spinner => default;
```

Comment:
In this case I don't believe this is the correct change to make here.

Making this a static property means that each access to the spinner effectively resets the internal state it has as all usage of it now gets a fresh value instead of the state being maintained for the lifetime of an instance of the rate limiter.

Either instead of `default` we should change the original code to use `new()`, and if that still raises warnings, then we should just explicitly suppress the CA1805 warning instead.